src/flatten.rs

```diff
@@ -6,9 +6,54 @@
 
 /// Extension trait for `Iterator<Item = Result<O, E>>` to selectively transform Oks and Errors.
 pub trait Flatten<O, E>: Sized {
+    /// [flatten](Iterator::flatten) `Err` values while leaving `Ok`-values as is.
```

Comment:
Isn't it the other way round?

Reply:
Of course, I must have been tired trying to get everything done yesterday. Will fix,